rules-reviewed/quarkus/springboot/springboot-data-jpa-to-quarkus.windup.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="springboot-jpa-to-quarkus"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset gives hints to migrate from SpringBoot JPA to Quarkus spring-jpa extension
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-xml,3.0.0.Final" />
+        </dependencies>
+        <sourceTechnology id="springboot" />
+        <targetTechnology id="quarkus" />
+    </metadata>
+    <rules>
+        <rule id="springboot-jpa-to-quarkus-00000">
+            <when>
+                <dependency groupId="org.springframework.data" artifactId="spring-data-jpa"/>
+            </when>
+            <perform>
+                <hint title="Replace the SpringBoot Data JPA artifact with Quarkus 'spring-data-jpa' extension" effort="1" category-id="mandatory">
+                    <message>
+                        Replace the SpringBoot JPA artifact with Quarkus `spring-data-jpa` extension
+
+                        Spring Data JPA is in spring-data-jpa artifact brought transitively by any `org.springframework.boot:spring-boot-*` dependency
+                        Add Quarkus dependency `io.quarkus:quarkus-spring-data-jpa-api`
```

Comment:
As I see in the documentation the dependency is :
<dependency>
    <groupId>io.quarkus</groupId>
    <artifactId>quarkus-spring-data-jpa</artifactId>
</dependency>
Not quarkus-spring-data-jpa-api

Reply:
that's correct! I changed it